src/corebluetooth.cs

```diff
@@ -34,9 +34,17 @@ interface CBCentralInitOptions {
 #endif
 	}
 
+	[iOS (10,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface CBManager {
+		[Export ("state", ArgumentSemantic.Assign)]
+		CBManagerState State { get; }
+	}
+
 	[Since (5,0)]
 	[Lion]
-	[BaseType (typeof (NSObject), Delegates=new[] {"WeakDelegate"}, Events = new[] { typeof (CBCentralManagerDelegate)})]
+	[BaseType (typeof (CBManager), Delegates=new[] {"WeakDelegate"}, Events = new[] { typeof (CBCentralManagerDelegate)})]
```

Comment:
Changing the `BaseType` can be tricky. You have to check if it's available in all OS (e.g. macOS) and it might also require a small hack to the registrar (to ignore it on older OS).